website_sale_product_total_seller_fields/views/website_sale_template.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="addition_fields_product" inherit_id="website_sale.products_description">
+        <xpath expr="//div[@itemprop='description']" position="replace">
+            <span t-field="product.main_seller_id"></span>
```

Comment:
small detail: add conditional to avoid rendering if expression is false
```xml
<t t-if="product.main_seller_id">
    <span t-field="product.main_seller_id"/>
</t>
```

Reply:
also for `product.total_with_vat_by_unit` :p